theme/index.pug

```diff
@@ -1,4 +1,4 @@
-- ts.sort((a, b) => a.name.localeCompare(b.name))
+- var toc = Object.keys(ts).sort((a, b) => a.localeCompare(b))
```

Comment:
well sure we need a sorted array for the table of contents. often both forms are needed, so i guess it's a tradeoff either way.